### PR TITLE
Azure CIS 4.5 Ensure that Azure Active Directory Admin is configured

### PR DESCRIPTION
**Pull request by XOmniverse**
Azure CIS 4.5 Ensure that Azure Active Directory Admin is configured

tested and working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,6 +257,7 @@ Reference
 ###### CIS Policies
 
 - [Azure Ensure SQL Server Auditing Enabled](./security/azure/sql_server_auditing/)
+- [Azure Ensure SQL Servers Have AD Admin Configured](./security/azure/sql_ad_admin/)
```
Comment (nia-vf1) [inline README.md:260]: Would maybe rename this to 'Azure Ensure SQL Server AD Admin Configured' for consistency

Comment (XOmniverse) [reply README.md]: Done

**Review by beagledeltasix — APPROVED** [review]
Reviewed and looks good. No changes from me.

**Review by nia-vf1 — DISMISSED** [review]
LGTM

**Review by nia-vf1 — APPROVED** [review]
Resolved conflict in the readme file

**Pull request merged**